global/css/_global.css

```diff
@@ -1063,5 +1063,77 @@ WYSIWYG formatting styles
 Single image/video component
 */
 .su-page-story-basic .su-play-button-icon svg {
-  @apply su-w-[40px] su-h-[40px] su-left-[20px] su-bottom-[20px] md:su-w-[60px] md:su-h-[60px] md:su-left-[26px] md:su-bottom-[26px];
+  @apply su-size-40 su-left-20 su-bottom-20 md:su-size-60 md:su-left-26 md:su-bottom-26;
 }
+
+/*
+Subscribe to Stanford Report Marketo form styles
+*/
+
+.marketo-sr-subscribe-form.mktoForm * {
+  @apply su-font-sans su-text-20;
+}
+
+.marketo-sr-subscribe-form.mktoForm .mktoFormRow {
+  @apply su-mb-50;
+}
+
+.marketo-sr-subscribe-form.mktoForm .mktoLabel {
+  @apply su-pb-2 su-font-semibold su-float-none su-text-18;
+}
+
+.marketo-sr-subscribe-form.mktoForm input.mktoField {
+  @apply su-input su-p-10 su-max-w-550 su-w-full;
+}
+
+.marketo-sr-subscribe-form.mktoForm input[type="text"],
+.marketo-sr-subscribe-form.mktoForm input[type="email"],
+.marketo-sr-subscribe-form.mktoForm textarea {
+  @apply su-border-black-20 dark:su-border-black-60 dark:su-bg-black-90 dark:su-text-white su-mt-6 su-rounded su-border su-border-b-2 su-border-b-black-70 dark:su-border-b-black-30 su-px-16 su-py-12 hocus:su-border-digital-blue-vivid hocus:su-border-b-digital-blue dark:hocus:su-border-digital-blue-light/80 dark:hocus:su-border-b-digital-blue-light focus:su-ring-4 focus:su-ring-digital-blue-vivid/20 dark:focus:su-ring-digital-blue-vivid/60 su-transition-colors;
+}
+
+/* Checkbox styles */
+.marketo-sr-subscribe-form.mktoForm .mktoCheckboxList {
+  @apply su-grid su-grid-cols-[auto_1fr] su-gap-x-14 su-gap-y-24 su-mt-20 su-p-0;
+}
+
+.marketo-sr-subscribe-form.mktoForm .mktoCheckboxList label {
+  @apply su-text-18 su-mt-01em su-leading-display su-cursor-pointer peer-hover:su-underline;
+}
+
+.marketo-sr-subscribe-form.mktoForm input[type="checkbox"] {
```

Comment:
FYI I removed all the pseudo element code for the checkboxes here, and it seems to work fine.